src/styles/theme.js

```diff
@@ -25,11 +25,12 @@ const theme = {
     default: '#93918F',
     headline: '#000000',
     link: {
+      default: ['#0087FF', '#1360A4'],
```

Comment:
Using an array here makes it hard to understand tbh. Only from reading the component styles it becomes clear that the second is the hover color.